sdk-core/src/main/java/care/data4life/sdk/network/NetworkingContract.kt

```diff
@@ -127,6 +127,19 @@ interface NetworkingContract {
         fun fromName(name: String?): Environment
     }
 
+    interface SearchTagsBuilder {
+        fun addOrTuple(tuple: List<String>): SearchTagsBuilder
+        fun seal(): SearchTags
+    }
+
+    interface SearchTags {
```

Comment:
what about a typealias?

Reply:
I'd like to avoid String here explicitly. Ideally the ApiService gets directly the Builder only with its build function exposed. However I tried to model that, which results in some overhead, which I like to avoid...so this i kind of a compromise.

Reply:
👍